src/app/playground/page.js

```diff
@@ -4,36 +4,69 @@ import { Paragraph, SubParagraph, SmallParagraph } from '@/components/shared/Typ
 import { PrimaryButton, SecondaryButton } from '@/components/shared/Typography/Buttons';
 import { NavbarLink } from '@/components/shared/Typography/Links';
 import { NavBar } from '@/components/NavBar/Nav';
+import Image from 'next/image';
 
 const Page = () => {
   return (
     <>
       <NavBar />
-      <div style={styles.mainContainer}>
-        <LogoText>innovision</LogoText>
-        <SubLogoText>24</SubLogoText>
-
-        <Heading1>ABOUT US</Heading1>
-        <Heading2>SubHeading</Heading2>
-
-        <Paragraph>
-          HackNITR 5.0 is a hackathon to encourage you to use your imagination and invention to
-          develop cutting-edge technology-based solutions to challenges in the real world. It was
-          put together by a large group of tech enthusiasts from Google Developer Student Club
-          (GDSC)
-        </Paragraph>
-        <SubParagraph>
-          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Delectus voluptas optio, aliquam
-          molestiae iure fuga animi beatae dolore magni odio. Odio eius vel vitae aut in quia.
-          Maiores, sit nobis.
-        </SubParagraph>
-        <SmallParagraph>
-          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Beatae impedit quidem qui nihil
-          sunt aspernatur eligendi? Dolorum, sequi, ratione impedit animi laboriosam quod
-          doloremque, ipsam quia voluptas qui perspiciatis earum.
-        </SmallParagraph>
+      <div className='relative z-10' style={styles.mainContainer}>
+        <div className='absolute -z-10 h-full w-full overflow-hidden flex mx-auto pt-20'>
+          <div
+            style={{
+              backgroundImage:
+                'linear-gradient(to bottom, rgba(197, 208, 255, 0.6) 0%, rgba(197, 208, 255, 0.4) 4%, rgba(21, 42, 126, 0.3) 50%,rgba(21, 42, 126, 0) 100%)',
+              boxShadow:
+                'inset 0 0 30px 30px rgba(11, 19, 42, 1), 0 0 50px 50px rgba(11, 19, 70, 0.5)',
+            }}
+            className='border border-[#070c23] drop-shadow-2xl rounded-full w-full h-full mt-64 '
+          ></div>
+        </div>
+        <section className='h-screen'>
```

Comment:
create a separate component in `components/hero/hero.jsx` and create styled comps in `components/hero/hero.jsx`